### DISCUSSION TITLE
Welcome to the Biostar Handbook Forum

### DISCUSSION THREAD

**Discussion by ialbert** [biostars/biostar-handbook-issues#183] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
 
  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.
-->

**Comment by rengang2020** [comment]
hello everyone, i am repeating the chip-seq analysis in the book (this section "Ming Tang’s guide to ChIP-Seq analysis" ).  this code does not work for me, "bowtie -p 10 --best --chunkmbs 320 /risapps/reference/bowtie1/hg19 -q YAP1.fastq -S ../results/bams/YAP1.sam"      ----Can anyone provide the linker for bw1 indexed hg19? thanks.

**Reply by rengang2020** [reply]
?? thanks
